cmd/init.go

```diff
@@ -51,53 +70,63 @@ func (h *Handler) initFromAccount(ctx context.Context, req *entity.CommandReques
 		return err
 	}
 
-	// Todo, prompt for environment
-	err = h.cfg.SetProjectConfigs(&entity.ProjectConfig{
-		Project:     project.Id,
-		Environment: project.Environments[0].Id,
-	})
+	err = h.cfg.SetProject(project.Id)
+	if err != nil {
+		return err
+	}
 
+	err = h.saveProjectAndEnvironment(ctx, project)
 	if err != nil {
-		return nil
+		return err
 	}
 
+	fmt.Printf("Connected to project %s 🎉\n", project.Name)
+
 	return nil
 }
 
-func (h *Handler) initFromID(ctx context.Context, req *entity.CommandRequest) error {
-	projectId, err := ui.PromptText("Enter your project id")
+func (h *Handler) saveProjectWithID(ctx context.Context, projectID string) error {
+	project, err := h.ctrl.GetProject(ctx, projectID)
 	if err != nil {
 		return err
 	}
 
-	project, err := h.ctrl.GetProject(ctx, projectId)
+	err = h.cfg.SetProject(project.Id)
 	if err != nil {
 		return err
 	}
 
-	err = h.cfg.SetProject(project.Id)
+	err = h.cfg.SetProject(projectID)
 	if err != nil {
 		return err
 	}
 
-	err = h.cfg.SetProject(projectId)
+	err = h.saveProjectAndEnvironment(ctx, project)
 	if err != nil {
 		return err
 	}
 
-	if len(project.Environments) > 0 {
-		err = h.cfg.SetEnvironment(project.Environments[0].Id)
-		if err != nil {
-			return err
-		}
-	}
-
 	fmt.Printf("Connected to project %s 🎉\n", project.Name)
 
 	return nil
 }
 
+func (h *Handler) initFromID(ctx context.Context, req *entity.CommandRequest) error {
+	projectID, err := ui.PromptText("Enter your project id")
+	if err != nil {
+		return err
+	}
+
+	return h.saveProjectWithID(ctx, projectID)
+}
+
 func (h *Handler) Init(ctx context.Context, req *entity.CommandRequest) error {
+	if len(req.Args) > 0 {
+		// projectID provided as argument
+		projectID := req.Args[0]
+		return h.saveProjectWithID(ctx, projectID)
```

Comment:
Can we keep these to following the common interface of taking a context + a CommandRequest?

projectId should be retrievable from req.Args[0]